docs/src/account/address.md

```diff
@@ -0,0 +1,63 @@
+# Address
```

Comment:
This is very well written, concise and clear. 

Can you maybe add how an address is linked to the `AccountID` to add even more clarity? One `AccountID` can have multiple addresses, so there is an n-to-1 mapping.

Reply:
Added a "Relationship to Identifiers" section that tries to explain this. Not sure if identifier is the best term here, so if you can think of a better one, please let me know.